lib/ops/opsservice/configure.go

```diff
@@ -230,7 +229,7 @@ func (s *site) configureExpandPackages(ctx context.Context, opCtx *operationCont
 			masterParams.sniHost = trustedCluster.GetSNIHost()
 		}
 		err = s.configurePlanetMasterSecrets(opCtx, masterParams)
-		if err != nil {
+		if err != nil && !trace.IsAlreadyExists(err) {
```

Comment:
Hmm so here and in all other places where you added this check (which is a bunch) - how can this happen? Don't the generated config and secret package names include timestamp?

I also wonder if silently swallowing this error may be misleading - if you request to generate the packages, shouldn't we rather upsert them than ignore if they already exist?

Reply:
Yes, they contain the timestamp which is pre-generated - to avoid generating the new package each time the function is re-run. If this generated a new package each time it was invoked, we could never properly generate a plan for an operation.

Do you have a suggestion on how to solve this issue in context of resumption - i.e. when a phase fails for some reason and then needs to re-execute the same steps incl. the generation of packages?